components/resource_control/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "resource_control"
```

Comment:
Could we keep this up with the PD by naming it as `resource_group_manager` or something? `resource_control` sounds too wide.

Reply:
Since it's based on the[ previous pr](https://github.com/tikv/tikv/pull/14001), I'll go to the last pr comment it

Reply:
Okay...I made this module related to pd_client, it makes more sense for me to change it